observability/micrometer/src/main/java/org/wildfly/extension/micrometer/MicrometerSubsystemSchema.java

```diff
@@ -5,37 +5,60 @@
 package org.wildfly.extension.micrometer;
 
 import static org.jboss.as.controller.PersistentResourceXMLDescription.factory;
+import static org.wildfly.extension.micrometer.MicrometerConfigurationConstants.OTLP_REGISTRY;
+import static org.wildfly.extension.micrometer.MicrometerConfigurationConstants.PROMETHEUS_REGISTRY;
+import static org.wildfly.extension.micrometer.prometheus.PrometheusRegistryDefinitionRegistrar.RESOURCE_REGISTRATION;
+
+import java.util.EnumSet;
+import java.util.Set;
 
 import org.jboss.as.controller.PersistentResourceXMLDescription;
 import org.jboss.as.controller.PersistentSubsystemSchema;
 import org.jboss.as.controller.SubsystemSchema;
 import org.jboss.as.controller.xml.VersionedNamespace;
+import org.jboss.as.version.Stability;
 import org.jboss.staxmapper.IntVersion;
+import org.wildfly.extension.micrometer.otlp.OtlpRegistryDefinitionRegistrar;
+import org.wildfly.extension.micrometer.prometheus.PrometheusRegistryDefinitionRegistrar;
 
 public enum MicrometerSubsystemSchema implements PersistentSubsystemSchema<MicrometerSubsystemSchema> {
-    VERSION_1_0(1, 0), // WildFly 28
-    VERSION_1_1(1, 1), // WildFly 29.0.0.Alpha1
+    VERSION_1_0(1, 0, Stability.DEFAULT), // WildFly 28
+    VERSION_1_1(1, 1, Stability.DEFAULT), // WildFly 29.0.0.Alpha1
+    VERSION_2_0(2, 0, Stability.DEFAULT), // WildFly 36
+    VERSION_2_0_COMMUNITY(2, 0, Stability.COMMUNITY) // WildFly 36
```

Comment:
This could be right, but I don't really understand it. I just don't see this pattern elsewhere. This will not block my approval though :) 

@pferraro Could you verify this is the correct pattern?